package.json

```diff
@@ -17,9 +17,9 @@
     "lint": "eslint .",
     "test": "vitest",
     "setup": "yarn && yarn locales:compile && yarn graphql:codegen",
-    "locales:extract": "NODE_ENV=development lingui extract --clean; ./scripts/strip-locale-line-numbers.sh -- tsx scripts/populate-aa.ts",
+    "locales:extract": "NODE_ENV=development lingui extract --clean; ./scripts/strip-locale-line-numbers.sh; tsx scripts/populate-aa.ts",
```

Comment:
Ah, my bad, instead of ";", it should be "&&" so that the whole command fails if the lingui fails.